recipes/eigency/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0a1" %}
```

Comment:
This is a pre-release version. We would like to have only releases con the main channel. Has there been a release meanwhile?